homeassistant/config_entries.py

```diff
@@ -459,7 +459,7 @@ async def setup_again(*_: Any) -> None:
 
             await self._async_process_on_unload()
             return
-        except Exception:  # pylint: disable=broad-except
+        except BaseException:  # pylint: disable=broad-except
```

Comment:
Should we instead do this?
```suggestion
        except (CancelledError, Exception):  # pylint: disable=broad-except
```